source/server-selection/server-selection.rst

```diff
@@ -521,6 +526,15 @@ Eligibility MUST be determined from ``tag_sets`` as follows:
 - If the ``tag_sets`` list is not empty and no tag set in the list matches any
   candidate server, no servers are eligible servers.
 
+hedge
+`````
+
+The read preference ``hedge`` parameter is an object that configures how the
```

Comment:
Let's say "document" instead of "object" here, since that's more clear when discussing BSON.